tests/unit/createRoute.ts

```diff
@@ -21,15 +19,15 @@ suite('createRoute', () => {
 			}
 		});
 
-		const selections = route.select({} as C, [], false, new UrlSearchParams());
+		const selections = route.select({} as Context, [], false, new UrlSearchParams());
 		assert.lengthOf(selections, 0);
 	});
 
 	test('guard() receives the context', () => {
-		const context: C = {};
-		let received: C = <any> undefined;
+		const context: Context = {};
+		let received: Context = <any> undefined;
```

Comment:
That's probably preferable over other hacks, yes, but I don't think we need to go and fix all tests to use that pattern. Tests are hard enough to maintain as it is.